app/scripts/components/common/chart/analysis/index.tsx

```diff
@@ -26,6 +26,7 @@ export default function AnalysisChartProps(props: AnalysisChartProps) {
       chartData={chartData}
       renderLegend={false}
       renderBrush={true}
+      renderExport={true}
```

Comment:
I have a question for you @danielfdsilva .

so I was wondering if it will be better to bring up `<ExportPNG />` component to this component, so ExportButton doesn't have to be affected by chart's wrapper style. so I did something like``

```
// chart/index.tsx
LineChartWithRef = React.forwardRef(RLineChart);
```
```
// chart/analysis/index.tsx
const chartRef = useRef(null);
<>
<LineChartWithRef ref={chartRef} ... />
<ExportPNG svgWrapperRef={chartRef} />
</>
```
But `ExportPNG` component doesn't seem to be able to get the update of `chartRef` 🤔 I couldn't really find a solution about it. I think current approach is ok, but I would like to hear how you would solve this?  (and if you think this is a better spot for `ExportPNG` component?)